### DISCUSSION TITLE
Cleaning up resources after a hack

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/RAG_Hack#123] [Q&A] [answered]
Hi, since all resources in Azure incur costs to keep them up and running, could you recommend how long our projects should remain active for you to be able to review it?

**Comment by multispark** [comment, marked as answer]
We are aiming to have projects completely judged by EOD PT September 23rd. However, we will give updates if this gets resolved sooner!

**Reply by multispark** [reply]
Deployed endpoints are not required! @YuriyMorozyuk95

**Reply by YuriyMorozyuk95** [reply]
I will let them up and running till 23d of September, if you would like to play and test it.

**Reply by YuriyMorozyuk95** [reply]
@multispark Hi, did you finish judging process, and can I clean up resources?

**Reply by multispark** [reply]
Hi Yuri, we finished judging for the categories that your project is a part of!
